libavpipe/src/avpipe_copy_mpegts.c

```diff
@@ -0,0 +1,389 @@
+/*
+ * Copy MPEGTS bypass encoder.
+ *
+ * Special code path to copy MPEGTS source media (MPEGTS or SRT input) to an alternate output.
+ *
+ * This could be generalized to copy any source encoding. MPEGTS is special in that
+ * all streams are muxed so there is only one output part (no separate video/audio)
+ */
+
+#include <libavutil/log.h>
+#include "libavutil/audio_fifo.h"
+#include <libswscale/swscale.h>
+#include <libavutil/imgutils.h>
+#include <libavutil/display.h>
+
+#include "avpipe_xc.h"
+#include "avpipe_utils.h"
+#include "avpipe_format.h"
+#include "avpipe_io.h"
+#include "avpipe_copy_mpegts.h"
+#include "elv_log.h"
+#include "elv_time.h"
+#include "url_parser.h"
+#include "avpipe_version.h"
+
+#include <stdio.h>
+#include <fcntl.h>
+#include <assert.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <sys/uio.h>
+#include <unistd.h>
+#include <stdlib.h>
+#include <pthread.h>
+
+
+int
+copy_mpegts_init(
```

Comment:
Do you anticipate needing to do things here?

Reply:
It had things initially but ended up not needed.  I removed it - we can add back if needed in the future.